src/main/java/uk/nhs/tis/trainee/notifications/dto/LtftUpdateEvent.java

```diff
@@ -46,6 +46,7 @@ public class LtftUpdateEvent {
   private ChangeDto change;
   private String state;
   private Instant timestamp;
+  private LftfStatusInfoDetailDto stateDetail;
```

Comment:
@DorisWongNHS I think I'm doing this a bit different to the way you are implementing https://github.com/Health-Education-England/tis-trainee-notifications/pull/273 - shall we agree on a common approach? 😁